src/OpalCompiler-Core/IRMethod.class.st

```diff
@@ -159,16 +159,6 @@ IRMethod >> forceLongForm [
 		
 ]
 
-{ #category : #debugging }
-IRMethod >> fullBlockInstructionForPC: aPC [
-	| pc |
-	"For a given PC, the actual instruction may start N bytes ahead."
-	pc := aPC.
-	[ pc >= 0 ] whileTrue: [ 
-		(self firstInstructionMatching: [:ir | ir bytecodeOffset = pc ]) ifNotNil: [:it |^it].
-		pc := pc - 1 ].
-]
```

Comment:
the OCBytecodeToASTCache iterates over all the IR and fills the cache. So something like this could be used for a regression test... but i would prefer to not have this method as part of the API of IRMethod.